tutorials/01-nextjs-clerk/app/globals.css

```
@@ -0,0 +1,22 @@
+@tailwind base;
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)